AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -65,6 +67,17 @@
  */
 @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
 
+/**
+ * @abstract Return the calculated layout.
+ *
+ * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
```

Comment:
What is the right way to do rounded corners of images or buttons if border radius is slow in objective c

On Mon, Jul 6, 2015 at 8:00 AM, Huy Nguyen <notifications@github.com>
wrote:

>> @@ -65,6 +67,17 @@
>>   */
>>  @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
>>  
>> +/**
>> + * @abstract Return the calculated layout.
>> + *
>> + * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
> Comment updated.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/facebook/AsyncDisplayKit/pull/468/files#r33942828